From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Karl Ding <kding@nvidia.com>
Date: Tue, 3 Feb 2026 21:09:14 +0000
Subject: [PATCH 169/241] NVIDIA: SAUCE: virt: tegra: Bump max IVC queues

The maximum number of IVC queues was increased in the PCT as part of
change Icf6930869a2050495c74dadbba502f530675d743 to reserve a range for
customer use.

Increase the PCT_MAX_NUM_IVC_QUEUES constant to match.

Bug 5703530

Change-Id: I661ead66c24edcc0bc3b6cca1a4662c55ad14102
Signed-off-by: Karl Ding <kding@nvidia.com>
Reviewed-on: https://git-master.nvidia.com/r/c/3rdparty/canonical/linux-noble/+/3541527
Reviewed-by: svcacv <svcacv@nvidia.com>
GVS: buildbot_gerritrpt <buildbot_gerritrpt@nvidia.com>
Reviewed-by: mobile promotions <svcmobile_promotions@nvidia.com>
Reviewed-by: Aparna Das <aparnad@nvidia.com>
Reviewed-by: Manish Bhardwaj <mbhardwaj@nvidia.com>
Reviewed-by: Matt Aamold <maamold@nvidia.com>
Tested-by: mobile promotions <svcmobile_promotions@nvidia.com>

diff --git a/include/soc/tegra/virt/tegra_hv.h b/include/soc/tegra/virt/tegra_hv.h
index 6fefb333bf09..27aecb68899c 100644
--- a/include/soc/tegra/virt/tegra_hv.h
+++ b/include/soc/tegra/virt/tegra_hv.h
@@ -23,7 +23,7 @@
 /** @brief Maximum Guest VM count */
 #define MAX_NUM_GUESTS		16U
 /** @brief The maximum number of IVC queues supported by the PCT. */
-#define PCT_MAX_NUM_IVC_QUEUES	512U
+#define PCT_MAX_NUM_IVC_QUEUES	800U
 /** @brief The maximum number of mempools supported by the PCT. */
 #define PCT_MAX_NUM_MEMPOOLS	120U
 
-- 
2.43.0

